src/runtime/lpython_builtin.py

```diff
@@ -619,6 +619,31 @@ def _lpython_str_capitalize(x: str) -> str:
     x = chr(val) + x[1:]
     return x
 
+@overload
```

Comment:
Hi, existing reference files get modified and new reference files get generated when you make an addition like this to the code base. You should run `./run_tests.py -u` to do the following.

Reply:
oh okay, I did that but that changed a lot of files so I asked on Zulip, thanks, I'll do that again.